cli/rotate.go

```diff
@@ -92,13 +94,19 @@ func RotateCommand(app *kingpin.Application, input RotateCommandInput) {
 		return
 	}
 
+	var iamUserName *string
+
+	// A username is needed for some IAM calls if the credentials have assumed a role
```

Comment:
Context here @mtibben is that for the root user this doesn't exist and needs to be null.